README.md

```diff
@@ -1 +1,43 @@
-# Enumerate
\ No newline at end of file
+# Enumerate
+
+> In this project we are learning how to use a module inside a Ruby class. For this we create a class MyList and a module MyEnumerable. Module MyEnumerable implements a subset of the functionality of Enumerable mixin.
+
+## Getting Started
+
+Install the gem file by executing:
+
+    $ bundle install
+
```

Comment:
- [ ] In your `README` file, Kindly update the "Getting Started" section. This should be an example of how you may give instructions on setting up your project locally. :+1: